help/configure-default-view.md

```diff
@@ -17,12 +17,13 @@ shortcut.
 
 ## Change default view
 
-Organization administrators can [set the default view for their
+Organization administrators can [configure the default view for their
 organization](/help/configure-default-new-user-settings) to
-[**Recent conversations**](/help/recent-conversations) or
-[**All messages**](/help/all-messages).
-**Recent conversations** is especially recommended for high-traffic
-organizations, and is configured by default.
+[**Recent conversations**](/help/recent-conversations),
+[**All messages**](/help/all-messages), or [**Inbox**](/help/inbox).
+The **Inbox** view works best if you regularly clear all unread messages
+in most streams you follow. Otherwise, **Recent conversations**
+works well in high-traffic organizations.
 
```

Comment:
I feel like we should say something about All messages here, too. Maybe:

> Otherwise, **Recent conversations** works well in high-traffic organizations. **All messages** is convenient for low-traffic organizations, or for skimming messages as they come in.